src/identity-providers/add/ReqAuthnConstraintsSettings.tsx

```diff
@@ -31,23 +31,23 @@ export const ReqAuthnConstraints = () => {
         fieldId="comparison"
       >
         <Controller
-          name="config.comparison"
+          name="config.authnContextComparisonType"
           defaultValue={comparisonValues[0]}
           control={control}
           render={({ onChange, value }) => (
             <Select
               toggleId="comparison"
               required
               direction="up"
-              onToggle={() => setSyncModeOpen(!syncModeOpen)}
+              onToggle={() => setComparisonOpen(!comparisonOpen)}
```

Comment:
The `onToggle` prop provides a value in the callback to set the open state so we don't have to control this ourselves.
```suggestion
              onToggle={(isExpanded) => setComparisonOpen(isExpanded)}
```

Reply:
always forget this one as well - fixed, ty